lib/openxml/has_properties.rb

```diff
@@ -142,6 +157,17 @@ def property_xml(xml)
       end
     end
 
+    def build_required_properties
+      required_properties.each do |prop|
+        public_send(:"#{prop}=", default_property_value_for(prop)) if respond_to? :"#{prop}="
```

Comment:
Would it work to have the default value be supplied as a keyword argument to `value_property` rather than for subclasses to have to override `default_property_value_for` and use a `case` statement or something like that to return default values for their required properties?

Reply:
Yeah, I like that a lot better. Made the change.